lib/rubocop/cop/sorbet/sigils/valid_sigil.rb

```diff
@@ -51,7 +51,7 @@ def autocorrect(_node)
         protected
 
         STRICTNESS_LEVELS = ["ignore", "false", "true", "strict", "strong"]
-        SIGIL_REGEX = /^\s*#\s+typed:(?:\s+([\w]+))?/
+        SIGIL_REGEX = /^\s*#\s+typed:(?:\s+([\S]+))?/
```

Comment:
While we are at it, can we also make these `\s` instances into `[[:blank:]]` since we don't actually want to match newlines, etc, just blank space:
```suggestion
        SIGIL_REGEX = /^[[:blank:]]*#[[:blank:]]+typed:(?:[[:blank:]]+([\S]+))?/
```